classes/class-wc-connect-order-item-meta.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Order_Item_Meta' ) ) {
+	class WC_Connect_Order_Item_Meta {
+		const PACKAGING = 'wc_connect_packages';
+
+		public static function delete_all_plugin_order_item_meta() {
+			if ( defined( 'WOOCOMMERCE_CONNECT_DEV_SERVER_URL' ) ) {
+				return;
+			}
+
+			global $wpdb;
+			$reflect = new ReflectionClass( get_called_class() );
+			$constants = $reflect->getConstants();
+			foreach ( $constants as $constantName => $constantValue ) {
+				$wpdb->query( "DELETE FROM {$wpdb->prefix}woocommerce_order_itemmeta WHERE meta_key='{$constantValue}'" );
```

Comment:
This should use [`$wpdb->prepare()`](https://developer.wordpress.org/reference/classes/wpdb/prepare/).